services/cargohold/src/CargoHold/API/Public.hs

```diff
@@ -57,6 +58,12 @@ servantSitemap =
     providerAPI = uploadAssetV3 @tag :<|> downloadAssetV3 @tag :<|> deleteAssetV3 @tag
     legacyAPI = legacyDownloadPlain :<|> legacyDownloadPlain :<|> legacyDownloadOtr
     qualifiedAPI = downloadAssetV4 :<|> deleteAssetV4
+    mainAPI =
+      renewTokenV3
+        :<|> deleteTokenV3
+        :<|> uploadAssetV3 @'UserPrincipalTag
+        :<|> downloadAssetV4
+        :<|> deleteAssetV4
```

Comment:
While at it, how about also renaming these handlers to remove old-style version suffices?

Reply:
I thought about it, but they're also used by the actual endpoints with v3 and v4 suffixes, so it kind of makes sense to leave it like this for now. We can normalise the naming once API v1 gets removed from the code.